torchvision/models/densenet.py

```diff
@@ -209,12 +209,12 @@ def __init__(self, growth_rate=32, block_config=(6, 12, 24, 16),
         # Official init from torch repo.
         for m in self.modules():
             if isinstance(m, nn.Conv2d):
-                nn.init.kaiming_normal(m.weight.data)
+                nn.init.kaiming_normal_(m.weight)
             elif isinstance(m, nn.BatchNorm2d):
-                m.weight.data.fill_(1)
-                m.bias.data.zero_()
+                m.weight.fill_(1)
```

Comment:
i think the `.data` is still needed. If not, the backward will backprop through the `fill_` and `zero_` calls